dags/gcs_bucket.py

```diff
@@ -24,3 +24,4 @@
 # GCS bucket for output
 BENCHMARK_OUTPUT_DIR = "gs://ml-auto-solutions/output/benchmark"
 XLML_OUTPUT_DIR = "gs://ml-auto-solutions/output/xlml"
+MAXTEXT_OUTPUT_DIR = "gs://ml-auto-solutions/output/maxtext/"
```

Comment:
extra `/` here that will make ` f" base_output_directory={gcs_bucket.MAXTEXT_OUTPUT_DIR}/nightly"` fail

Reply:
Good catch, Thanks!